docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -0,0 +1,147 @@
+# How to use cPouta for Remote Desktop
```

Comment:
title is cPouta, description is c- and ePouta

Reply:
fixed